adr/ADR-49.md

```diff
@@ -226,31 +226,38 @@ In this case a dedicated light weight Counter client to begin with would be adde
 > This is just an idea at present, the client will be fleshed out later in the development cycle.
 
 ```go
-// Creates a new counter abstraction bound to a Stream
-func NewCounter(stream jetstream.Stream) (Counter, error)
-
-// Parses a messages received from a consumer into a counter entry
-func ParseMessage(msg jetstream.Msg) (Entry, error)
-
-// Entry holds helpers for parsing the values in counters
-type Entry interface {
-	// Value parses the value as a big.Int
-	Value() (big.Int, error)
-	
-// The messages that contributed to the Value
-	Messages() []jetstream.Msg
-}
+``package counters
+
+import "math/big"
 
-// Increments and loads 
 type Counter interface {
-    // Increments a counter by delta
-    func Increment(subject string, value int64) (Entry, error)
-	
-    // Loads the value from the stream subject, options to supply a specific seq/time for example
-    func Load(subject string, opts ...LoadOption) (Entry, error)
-
-    // Loads a group of subjects from the same stream using a direct batch get, performs the
-    // calculation to combine these numbers client side
-    func LoadMulti(subjects []string, opts ...LoadOptions) ([]Entry, error)
+     /// Adds or subtracts a value from the counter, returning the new value.
```

Comment:
spacing isnt right for these comments vs the functions, also weird with 3 slashes

Reply:
fixed